hhast-lint.json

```diff
@@ -1,5 +1,5 @@
 {
-  "roots": [ "src/", "codegen/", "tests/" ],
+  "roots": [ "src/", "codegen/", "tests/", "test-data/" ],
```

Comment:
Please remove this and add a test that invokes this and makes sure it's not linted

Reply:
Didn't see this earlier, resolved.